internal/repository/master_index.go

```diff
@@ -264,49 +264,63 @@ func (mi *MasterIndex) MergeFinalIndexes() {
 	mi.idx = newIdx
 }
 
-// RebuildIndex combines all known indexes to a new index, leaving out any
+// RebuildIndex saves all known indexes to index files, leaving out any
 // packs whose ID is contained in packBlacklist. The new index contains the IDs
-// of all known indexes in the "supersedes" field.
-func (mi *MasterIndex) RebuildIndex(packBlacklist restic.IDSet) (*Index, error) {
+// of all known indexes in the "supersedes" field. The IDs are also returned in
+// the IDSet obsolete
+func (mi *MasterIndex) RebuildIndex(ctx context.Context, repo restic.Repository, packBlacklist restic.IDSet) (obsolete restic.IDSet, err error) {
```

Comment:
I think this method should now be called `Save`.

Reply:
changed.